firmware_configurations/klipper/TechnoPhreak/BTT_SKR_v2/printer.cfg

```diff
@@ -0,0 +1,608 @@
+# 	Displays
+#####################################################################
+
+## 	Uncomment the display that you have. Display connects to Z MCU
+#--------------------------------------------------------------------
+
+#[display]
+##	RepRapDiscount 128x64 Full Graphic Smart Controller
+#lcd_type: st7920
+#cs_pin: z:PB9
+#sclk_pin: z:PE10
+#sid_pin: z:PB1
+#menu_timeout: 40
+#encoder_pins: ^z:PE7, ^z:PB2 
+#click_pin: ^!z:PB0
+
+#[output_pin beeper]
+#pin: z:PC5
+
+#--------------------------------------------------------------------
+
+#[display]
+##	mini12864 LCD Display
+#lcd_type: uc1701
+#cs_pin: z:PB1
+#a0_pin: z:PB9
+#encoder_pins: ^z:PB2,^z:PE7
+#click_pin: ^!z:PB0
+#contrast: 63
+
+#[neopixel fysetc_mini12864]
+##	To control Neopixel RGB in mini12864 display
+#pin: z:PE11
+#chain_count: 3
+#initial_RED: 0.1
+#initial_GREEN: 0.5
+#initial_BLUE: 0.0
+#color_order: RGB
+
+##	Set RGB values on boot up for each Neopixel. 
+##	Index 1 = display, Index 2 and 3 = Knob
+#[delayed_gcode setdisplayneopixel]
+#initial_duration: 1
+#gcode:
+#        SET_LED LED=fysetc_mini12864 RED=1 GREEN=1 BLUE=1 INDEX=1 TRANSMIT=0
+#        SET_LED LED=fysetc_mini12864 RED=1 GREEN=0 BLUE=0 INDEX=2 TRANSMIT=0
+#        SET_LED LED=fysetc_mini12864 RED=1 GREEN=0 BLUE=0 INDEX=3 
+
+#--------------------------------------------------------------------
+
+
+#####################################################################
+# 	Macros
+#####################################################################
+
+[gcode_macro G32]
+gcode:
+    BED_MESH_CLEAR
+    G28
+    QUAD_GANTRY_LEVEL
+    G28
+    ##	Uncomment for for your size printer:
+    #--------------------------------------------------------------------
+    ##	Uncomment for 250mm build
+    #G0 X125 Y125 Z30 F3600
+    
+    ##	Uncomment for 300 build
+    #G0 X150 Y150 Z30 F3600
+    
+    ##	Uncomment for 350mm build
+    #G0 X175 Y175 Z30 F3600
+    #--------------------------------------------------------------------
+   
+[gcode_macro PRINT_START]
+#   Use PRINT_START for the slicer starting script - please customise for your slicer of choice
+gcode:
+    G32                            ; home all axes
+    G1 Z20 F3000                   ; move nozzle away from bed
+   
+
+[gcode_macro PRINT_END]
+#   Use PRINT_END for the slicer ending script - please customise for your slicer of choice
+gcode:
+    M400                           ; wait for buffer to clear
+    G92 E0                         ; zero the extruder
+    G1 E-10.0 F3600                ; retract filament
+    G91                            ; relative positioning
+    G0 Z1.00 X20.0 Y20.0 F20000    ; move nozzle to remove stringing
+    TURN_OFF_HEATERS
+    M107                           ; turn off fan
+    G1 Z2 F3000                    ; move nozzle up 2mm
+    G90                            ; absolute positioning
+    G0  X125 Y250 F3600            ; park nozzle at rear
+    BED_MESH_CLEAR
+    
+## 	Thermistor Types

```

Comment:
remove these lines since they're out of date. Users can look at the link we added above to get the up to date thermistor types. make sure there's an empty line at the end of the file.

Reply:
Should be good now